### DISCUSSION TITLE
Input in spanish writes symbols instead of letters

### DISCUSSION THREAD

**Discussion by ttymothy** [davatorium/rofi#1644] [(Possible) Issues] [answered]
Hi, I'm not sure whether this is an issue, so I'm trying with a discussion first.

I have been using rofi for months without issue, and it has become my favourite launcher. However, in the last week something weird has started happening: when the keyboard input language is set to Spanish (Spain), writing in a rofi window results in gibberish being written (symbols).

* Rofi version: 1.7.3 (I have not updated recently, so I think that it is a dependency's fault)
* OS and desktop: Arch Linux (updated to latest packages), the issue appears in both KDE (X11) and Sway (Wayland).

How to reproduce:
1. Install in Arch Linux from the official repositories (`pacman -S rofi`)
2. Set the language to Spanish (`setxkbmap es`)
3. Run rofi (`rofi -show run`)
4. Write something (for example "hello world"), and see those letters. Instead, I see:
![imagen](https://user-images.githubusercontent.com/81260861/173130280-5fc00117-e5e3-4444-8276-60f7d9f83c69.png)

The symbols written are not random, but I cannot remember how to access that keyboard layer in Linux.

As I said before, this problem has appeared recently, and I've been using the same version of rofi for a long time without issues, so this must be a problem with rofi's deps. I have also checked that running rofi without my config (`rofi -config /dev/null -show run`) doesn't solve the issue.

**Comment by DaveDavenport** [comment, marked as answer]
feels awfully similar too https://github.com/davatorium/rofi/issues/1642
From rofi side nothing changed in keyboard handling recently.

**Reply by ttymothy** [reply]
I feel dumb now for not being able to find that issue... Thanks :D

**Comment by DaveDavenport** [comment]
p.s. there is a (documented) `-no-config` option.
